### DISCUSSION TITLE
Une commande pour Donjon

### DISCUSSION THREAD

**Discussion by Kuro2k7** [Crownicles/Crownicles#832] [Idées acceptées / déjà implémentées]
commande pour donjon avec ses pote ou membre de sa guilde , 
1 donjon par jour , et faut être minimu lvl 20

**Comment by QQtin3** [comment]
![image-5.png](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/56274541/d612af3c-2d4e-45b2-ae01-db1fe0526a70)

**Comment by BastLast** [comment]
cette feature est basiquement l'île pve
